modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -6,10 +6,10 @@
 [id="jt-examples-of-common-use-cases_{context}"]
 = Examples of common use cases
 
-Both Jenkins and OpenShift Pipelines offer capabilities for common CI/CD use cases, such as:
+Both Jenkins and Tekton offer capabilities for common CI/CD use cases, such as:
```

Comment:
Any reason for replacing `Openshift Pipelines` with `Tekton`? 
cc: @vdemeester @ppitonak

Reply:
Yup, I see no reason to change it and linked Bugzilla doesn't mention Pipelines/Tekton at all.

Reply:
The scope of this bz is to replace **plugin with plug-in** as per the IBM style guide. There is no change to the term OpenShift pipelines/Tekton.

Reply:
@Amrita42, this topic's used in a different assembly: https://deploy-preview-43918--osdocs.netlify.app/openshift-enterprise/latest/cicd/jenkins/migrating-from-jenkins-to-openshift-pipelines.html#jt-examples-of-common-use-cases_migrating-from-jenkins-to-openshift-pipelines

I suggest reverting this change and stick to just the plug-in and parameter changes.

Reply:
I think this change would have happened during resolving a merge conflict. The scope is to change plugin and paramter . 
Let me just create a new PR and close this one.